src/Apps/Order/Routes/Offer/index.tsx

```diff
@@ -0,0 +1,226 @@
+import { Button, Flex, Sans, Spacer } from "@artsy/palette"
+import { Offer_order } from "__generated__/Offer_order.graphql"
+import { Helper } from "Apps/Order/Components/Helper"
+import { TransactionSummaryFragmentContainer as TransactionSummary } from "Apps/Order/Components/TransactionSummary"
+import { TwoColumnLayout } from "Apps/Order/Components/TwoColumnLayout"
+import { track } from "Artsy/Analytics"
+import { ContextConsumer, Mediator } from "Artsy/SystemContext"
+import { Input } from "Components/Input"
+import { ErrorModal } from "Components/Modal/ErrorModal"
+import { Router } from "found"
+import React, { Component } from "react"
+import { createFragmentContainer, graphql, RelayProp } from "react-relay"
+import { Col, Row } from "Styleguide/Elements/Grid"
+import { HorizontalPadding } from "Styleguide/Utils/HorizontalPadding"
+import { get } from "Utils/get"
+import { Responsive } from "Utils/Responsive"
+import { OrderStepper } from "../../Components/OrderStepper"
+
+export interface OfferProps {
+  order: Offer_order
+  mediator: Mediator
+  relay?: RelayProp
+  router: Router
+}
+
+export interface OfferState {
+  offerValue: number | null
+  isCommittingMutation: boolean
+  isErrorModalOpen: boolean
+  errorModalTitle: string
+  errorModalMessage: string
+}
+
+function formatMoney(money: number | undefined): string | null {
+  if (money == null) return null
+
+  const [dollars, cents] = money.toFixed(2).split(".")
+  const dollarDigits = dollars.split("")
+
+  for (let i = dollarDigits.length - 3; i > 0; i -= 3) {
+    dollarDigits.splice(i, 0, ",")
+  }
+
+  const formattedDollars = dollarDigits.join("")
+
+  if (cents === "00") {
+    return `$${formattedDollars}`
```

Comment:
We should at least add a TODO here about supporting an actual currency if that's known.

Reply:
You can actually use `toLocaleString` to format a number into it's currency format. See (https://github.com/artsy/reaction/blob/master/src/Styleguide/Components/PriceRange.tsx#L50)

Reply:
Aw nice, was not aware of this.